include/gc/Analysis/MatmulConfigAnalysis.h

```diff
@@ -47,15 +47,67 @@ inline SmallVector<unsigned> extractDimTypeIdx(ArrayRef<DimType> tyList,
   return idxList;
 }
 
+inline void getDimTypeFromIterators(linalg::LinalgOp linalgOp,
+                                    SmallVectorImpl<DimType> &dimTypes) {
+  SmallVector<utils::IteratorType> iteratorTypes =
```

Comment:
Can you explicitly specify `mlir::utils::IteratorType` here?

Reply:
add `mlir` namespace here